src/pages/StatusPage.vue

```diff
@@ -708,21 +715,21 @@ export default {
             this.$root.publicGroupList = res.data.publicGroupList;
 
             this.loading = false;
+
+            // Configure auto-refresh loop
+            this.updateHeartbeatList();
```

Comment:
This line (720) being moved up here means that the status page will only fetch the heartbeat data after the status page data is loaded, which will add a round-trip delay. I don't think it's necessary?

Reply:
You are rigth. I move back this where it was